test/Dialect/FIRRTL/SFCTests/GrandCentralInterfaces/DropCompanion.fir

```diff
@@ -0,0 +1,65 @@
+; RUN: firtool %s --grand-central-companion-mode=drop | FileCheck %s --check-prefix CHECK
+
+circuit Foo : %[[
+  {
+    "class": "sifive.enterprise.grandcentral.GrandCentralView$SerializedViewAnnotation",
+    "name": "bar_view",
+    "companion": "~Foo|Companion",
+    "parent": "~Foo|Foo",
+    "view": {
+      "class": "sifive.enterprise.grandcentral.AugmentedBundleType",
+      "defName": "Bar_View",
+      "elements": [
+        {
+          "name": "bar_in",
+          "description": "bar in",
+          "tpe": {
+            "class": "sifive.enterprise.grandcentral.AugmentedGroundType",
+            "ref": {
+              "circuit": "Foo",
+              "module": "Foo",
+              "path": [],
+              "ref": "a",
+              "component": []
+            },
+            "tpe": {
+              "class": "sifive.enterprise.grandcentral.GrandCentralView$UnknownGroundType$"
+            }
+          }
+        }
+      ]
+    }
+  },
+  {
+    "class": "sifive.enterprise.grandcentral.ExtractGrandCentralAnnotation",
+    "directory": "Wire/firrtl/gct",
+    "filename": "Wire/firrtl/bindings.sv"
+  },
+  {
+    "class": "sifive.enterprise.grandcentral.GrandCentralHierarchyFileAnnotation",
+    "filename": "gct.yaml"
+  }
+]]
+  module Companion :
+    input clock: Clock
+    input a: UInt<1>
+
+    assert(clock, a, UInt<1>(1), "hello")
```

Comment:
This should also be checking that this module doesn't exist. 

Having a unified test that checks all three behaviors via three `RUN` lines would be simpler to grok and maintain.